docs/user-guide/python-coprocessor/overview.md

```diff
@@ -12,3 +12,31 @@ We think the python coprocessor in GreptimeDB is a perfect replacement for store
 * [Builtin Module](./builtins.md)
 * [Third-parties Libs](./third-parties.md)
 * [FAQ](./faq.md)
+
+# Note:
+The python coprocessor is still in the experimental stage, and the API is subject to change. 
+
+It also comes with some shenanigans, you need to set up the path of correct python shared library, this could a bit tricky, usually you just need to install `python-dev` package(Like on most debian-based system), but on MacOs you might need to set up correct soft link to `Library/Frameworks/Python.framework` if you are using homebrew to install python.
+
+So if you can't run your binary on MacOs because of link error, you can try to fix it like this(This is a temporary solution, we will fix it in the future):
```

Comment:
```suggestion
So if you can't run your binary on macOS because of link error, you can try to fix it like this (This is a temporary solution, we will fix it in the future):
```